WalletWasabi.Fluent/Views/Wallets/Home/History/HistoryTable.axaml

```diff
@@ -267,10 +267,23 @@
 
       </TreeDataGrid.Styles>
     </TreeDataGrid>
-    <Panel IsVisible="{Binding IsTransactionHistoryEmpty}">
+    <Panel IsVisible="{Binding !IsTransactionHistoryLoaded}">
       <c:HistoryPlaceholderPanel Margin="{StaticResource HistoryTablePlaceholderPanelMargin}"
                                  RowHeight="{StaticResource HistoryTableRowHeight}" />
       <Panel Background="{DynamicResource RegionColor}" Opacity="0.4" />
     </Panel>
+    <Panel IsVisible="{Binding IsTransactionHistoryLoadedAndEmpty}"
+           Margin="{StaticResource HistoryTablePlaceholderPanelMargin}">
+      <c:HistoryPlaceholderPanel RowHeight="{StaticResource HistoryTableRowHeight}" />
+      <Panel Background="{DynamicResource RegionColor}" Opacity="0.85" />
+      <Grid RowDefinitions="26,1*,1.5*">
+        <Panel Grid.Row="1">
+          <c:InfoMessage  VerticalAlignment="Center"
+                          HorizontalAlignment="Center">
+            <TextBlock Text="Transaction history is empty" />
+          </c:InfoMessage>
+        </Panel>
+      </Grid>
```

Comment:
I couldn't manage to make it nice visually on big screens without the grid

<details>
<summary>Option 1: Quite too low</summary>

![WalletWasabi Fluent Desktop_PDj29jWhoN](https://user-images.githubusercontent.com/16029533/185910153-73464642-f32d-4ec4-9579-84d03e8df4d6.png)
</details>

<details>
<summary>Option 2: Quite too high</summary>

![WalletWasabi Fluent Desktop_yVsmu8Q85y](https://user-images.githubusercontent.com/16029533/185910947-5c4af4c4-59aa-4d1f-b49a-a37548fd2182.png)
</details>

Maybe there is another way to make it responsive (without centering as the dummy lines don't go all the way bottom so that looks a bit weird) that I'm unaware of?
Otherwise you are right I think performance should prevale and Option 1 implemented

Reply:
Try to use `Margin` if that doesn't work, I will take a look.